src/components/AgreementSections/CoveredEResourcesList.js

```diff
@@ -73,7 +73,7 @@ export default class CoveredEResourcesList extends React.Component {
     },
     platform: e => get(e._object, 'pti.platform.name', '-'),
     package: e => get(e._object, 'pkg.name', '-'),
-    coverage: e => <CoverageStatements statements={e.coverage} />,
+    coverage:  e => <Coverage pci={e._object} />,
```

Comment:
Careful with the spacing
```suggestion
    coverage: e => <Coverage pci={e._object} />,
```

Reply:
Are you sure about this? It looks like you're now showing the base coverage and not custom coverage if it exists.